app/livechat/client/views/app/livechatCustomFields.js

```diff
@@ -1,20 +1,26 @@
 import { Meteor } from 'meteor/meteor';
 import { FlowRouter } from 'meteor/kadira:flow-router';
 import { Template } from 'meteor/templating';
+import { ReactiveVar } from 'meteor/reactive-var';
 
 import { modal } from '../../../../ui-utils';
 import { t, handleError } from '../../../../utils';
```

Comment:
this can be changed to import from `../../../../utils/client` and then include `APIClient` to imports list.